### DISCUSSION TITLE
Error when verify installation

### DISCUSSION THREAD

**Discussion by MukeshBabuPJ** [The-OpenROAD-Project/OpenROAD-flow-scripts#1534] [General]
I am installing OpenROAD using this documentation 
https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html
After running **make** command in **flow** directory
I got error in cts.tcl no 110 couldn't execute "eqy"
Here's a screenshot of that error
![Capture](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/114819520/b3870e19-b0cc-48a6-b3f5-351c6e0338c0)

**Comment by maliberty** [comment]
@vvbandeira please respond

**Reply by vijayank88** [reply]
Follow-up this issue: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues/1513
